### PR TITLE
convert ForgotPassword.css to ForgotPassword.module.css, improve set new password page

### PR DESCRIPTION
**Pull request by parthmanhas**
image

### CODE REVIEW

File: frontend/src/scenes/login/ForgotPassword.module.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+@import url(../../styles/variables.css);
+
+.form-group-2 {
```
Comment (erenfn) [inline frontend/src/scenes/login/ForgotPassword.module.css:3]: this doesn't seem to be used: .form-group-2

File: frontend/src/scenes/login/ForgotPassword.module.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+.form-group label {
+  display: block;
+  margin-bottom: 5px;
+  font-size: 13px;
+  color: var(--main-text-color);
+  font-weight: 500;
+}
+
+
+.login-container {
```
Comment (erenfn) [inline frontend/src/scenes/login/ForgotPassword.module.css:18]: delete

`font-family: 'Inter', sans-serif;`
  
in login container (also delete this in Login.css)

File: frontend/src/scenes/login/ForgotPassword.module.css [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+  margin-bottom: 30px;
+  margin-top: 20px;
+  font-size: 16px;
+  font-weight: 600;
+  line-height: 38px;
+}
+
+
+.login-container h3 {
+  /* font-size: 16px; */
```
Comment (erenfn) [inline frontend/src/scenes/login/ForgotPassword.module.css:45]: Delete commented out lines

File: frontend/src/scenes/login/ForgotPassword.module.css [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  line-height: 24px;
+}
+
+
+/* Forgot Password CSS */
+
+.back-to-login-button {
+  width: 100%;
+  padding: 10px;
+  margin-top: 0px;
```
Comment (erenfn) [inline frontend/src/scenes/login/ForgotPassword.module.css:61]: delete margin-top: 0px;

**Review by erenfn — COMMENTED** [review]
- Some styles overlap with the styles in Login.css. You can also use login.css style some components.
- Additionally, delete 

```
    /* font-weight: 400; */
    /* font-size: 40px;
```
from logo-container.

File: frontend/src/scenes/login/ForgotPassword.module.css [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  display: block;
+  margin-bottom: 5px;
+  font-size: 13px;
+  color: var(--main-text-color);
+  font-weight: 500;
+}
+
+
+.login-container {
+  width: 100%;
```
Comment (uparkalau) [inline frontend/src/scenes/login/ForgotPassword.module.css:19]: Can we use flex here as well

File: frontend/src/scenes/login/ForgotPassword.module.css [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+.error-message {
+  color: #D92D20;
+  font-size: 13px;
+  font-weight: 400;
+  margin-bottom: 1rem;
+  margin-top: 0.25rem;
+}
+
+.reset-password-button {
```
Comment (uparkalau) [inline frontend/src/scenes/login/ForgotPassword.module.css:82]: Use flex here for elements position

File: frontend/src/scenes/login/ForgotPassword.module.css (hunk 5 above)
Comment (erenfn) [reply frontend/src/scenes/login/ForgotPassword.module.css:19]: I think flex is already inherited from `html, body, #root`

Comment (erenfn) [reply frontend/src/scenes/login/ForgotPassword.module.css:19]: We can add it, but functionality wise I didn't see any difference

File: frontend/src/scenes/login/ForgotPassword.module.css (hunk 6 above)
Comment (erenfn) [reply frontend/src/scenes/login/ForgotPassword.module.css:82]: I also don't think we need flex here, this class is just for the styling of the button. Adding

```
display: flex;
align-items: center;
justify-content: center;
```

Doesn't change anything

File: frontend/src/scenes/login/ForgotPassword.module.css (hunk 5 above)
Comment (uparkalau) [reply frontend/src/scenes/login/ForgotPassword.module.css:19]: Well if you don't see difference, just use flex.

File: frontend/src/scenes/login/ForgotPassword.module.css (hunk 2 above)
Comment (parthmanhas) [reply frontend/src/scenes/login/ForgotPassword.module.css]: Fixed

File: frontend/src/scenes/login/ForgotPassword.module.css (hunk 3 above)
Comment (parthmanhas) [reply frontend/src/scenes/login/ForgotPassword.module.css]: Fixed

File: frontend/src/scenes/login/ForgotPassword.module.css (hunk 4 above)
Comment (parthmanhas) [reply frontend/src/scenes/login/ForgotPassword.module.css]: removed this in Login.module.css

File: frontend/src/scenes/login/ForgotPassword.module.css (hunk 5 above)
Comment (parthmanhas) [reply frontend/src/scenes/login/ForgotPassword.module.css]: Used flex in Login Container

**Comment by parthmanhas** [comment]
added Set New Password Page
image

**Review by erenfn — APPROVED** [review]

**Pull request merged**